web/views.py

```
@@ -4644,77 +4644,113 @@ def delete_team_goal(request, goal_id):
+                    messages.success(request, f"{student.get_full_name()} has been enrolled in the course.")
+                    
+                    # Send enrollment notifications
+                    send_enrollment_confirmation(enrollment)
+                    notify_teacher_new_enrollment(enrollment)
+                    
+                    # Send enrollment notification email to existing student
+                    context = {
+                        "student": student,
+                        "course": course,
+                        "teacher": request.user,
+                        "is_existing_user": True
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)